app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="recipes-container">
+<h1>Public Recipes</h1>
+
+<ul class="public_recipes_container">
+  <% @public_recipes.each do |public_recipe| %>
+    <%= link_to recipe_path(id: public_recipe.id) do%>
+      <li>
+        <div>
+          <h2><%= public_recipe.name %></h2>
+          <span><%= public_recipe.user.name %></span>
+        </div>
+        <div>
+          <p>Total food items: <%= public_recipe.recipe_foods.count %></p>
+          <p>Total Price
+            <% total_price = 0 %>
+            <% public_recipe.recipe_foods.each do |recipe_food| %>
+            <% total_price += recipe_food.quantity * recipe_food.food.price %>
+            <% end %>
+            <%= total_price %>
+          </p>
+        </div>
+      </li>
+    <% end %>
+  <% end %>
+</ul>
+
```

Comment:
- [ ] @team great work on your public recipe page, however, the layout design doesn't match the wireframe given to check the comparison below for a better understanding:


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/222666540-47f8d548-d672-4f62-914d-47fb1648b508.png)  |  ![image](https://user-images.githubusercontent.com/61408860/222666641-76b10629-1256-4a74-9d2b-804114c9fd8f.png)